base/applications/rapps/installed.cpp

```diff
@@ -45,30 +45,42 @@ void CInstalledApplicationInfo::EnsureDetailsLoaded()
         GetApplicationRegString(L"Contact", szContact);
         GetApplicationRegString(L"URLUpdateInfo", szURLUpdateInfo);
         GetApplicationRegString(L"URLInfoAbout", szURLInfoAbout);
-        if (GetApplicationRegString(L"InstallDate", szInstallDate) == FALSE)
+
+        DWORD dwInstallTimeStamp;
+        SYSTEMTIME InstallLocalTime;
+        if (GetApplicationRegString(L"InstallDate", szInstallDate))
         {
-            // It might be a DWORD (Unix timestamp). try again.
-            DWORD dwInstallTimeStamp;
-            if (GetApplicationRegDword(L"InstallDate", &dwInstallTimeStamp))
+            ZeroMemory(&InstallLocalTime, sizeof(InstallLocalTime));
+            // Check if we have 8 characters to parse the datetime.
+            // Maybe other formats exist as well?
+            if (szInstallDate.GetLength() == 8)
```

Comment:
This small addition fixes it for me:
```suggestion
            szInstallDate = szInstallDate.Trim();
            if (szInstallDate.GetLength() == 8)
```